addons/web_editor/static/src/components/media_dialog/image_selector.xml

```diff
@@ -40,7 +40,7 @@
                         onImageClick="() => this.onClickAttachment(attachment)"/>
                 </t>
             </t>
-            <t t-if="['all', 'media-library'].includes(state.searchService)">
+            <t t-if="['all', 'media-library'].includes(state.searchService)" id="o_we_media_library_images">
```

Comment:
Nitpicking: Not ideal to have an `id` on a `</t>` element but in this case it is fine, more robust as you said. But just move the `id` as first attribute.